dowhy/gcm/stats.py

```diff
@@ -211,7 +211,7 @@ def estimate_ftest_pvalue(X_training_a: np.ndarray,
         ssr_b = np.sum(
             (Y_test - LinearRegression().fit(X_training_b, Y_training).predict(X_test_b)) ** 2)
     else:
-        ssr_b = np.sum((Y_test - np.mean(Y_test)) ** 2)
+        ssr_b = np.sum((Y_test - np.mean(Y_training)) ** 2)
```

Comment:
But this is not the error of the model (without inputs) though. Why does this make sense? Can you also point to any articles where this is advocated or advised?

Reply:
This else condition is the scenario where we don't have input features, i.e. empty conditioning set. Therefore we have: ```E[Y | {}] = E[Y]```. Here, ```Y``` is based on ```Y_training```. Therefore, the prediction model we are looking here at is ```f({}) = E[Y | {}] = E[Y]``` and, thus, the squared error in the test data is ```sum_i (y_i - f({}))**2 = sum_i (y_i - E[Y])**2```